### ISSUE TITLE
RTSP with ERROR] no server reply 8.7-dev

### ISSUE THREAD

**Issue by francisuk1989** [vanhauser-thc/thc-hydra#282]
hydra -l admin -P "testpws.txt" -o "results.txt" -M localips.txt rtsp

Hydra (http://www.thc.org/thc-hydra) starting at 2017-11-13 23:47:45
[DATA] max 1 task per 50 servers, overall 50 tasks, 101 login tries (l:1/p:101), ~101 tries per task
[DATA] attacking rtsp://(50 targets):554/
[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply[ERROR] no server reply^CThe session file ./hydra.restore was written. Type "hydra -R" to resume session.

I am using the following command with ubuntu 16.04.3 xenial on a x64 machine, Also have installed the requirements from this github, i have also checked of the device is online what it is.

**Comment by vanhauser-thc** [comment]
have you checked that the hosts are open on TCP port 554 and they are RTSP servers?
you can do a quick check with nmap
nmap -sSVC -n -p 554 -iL localips.txt

**Comment by lansalot** [comment]
Getting the same. Device is online, and nmap reports the device is responding and reachable, but Hydra spams with [ERRROR] no server reply.

Linux hostname 4.16.0-trunk-amd64 #1 SMP Debian 4.16-1~exp1 (2018-04-08) x86_64 GNU/Linux
hydra v8.6

Starting Nmap 7.70 ( https://nmap.org ) at 2018-04-16 10:56 GMT Daylight Time
Nmap scan report for (x.x.x.x)
Host is up (0.0060s latency).

PORT    STATE SERVICE VERSION
554/tcp open  rtsp    Avigilon ONVIF camera rtspd 2.6.0.100
Service Info: Device: webcam

**Comment by vanhauser-thc** [comment]
you can add the -d switch to hydra, which enables debugging.
lots of it will bit not important for you, but the CONNECT and RECV lines should be useful.

**Comment by vanhauser-thc** [comment]
also - wireshark/tcpdump :)

**Comment by lansalot** [comment]
Got a debug output, but it the responses made me curious in that there didn't appear to be any..

So I probed the source device a little harder - it appears to be some sort of kiosk PC that is no longer responding to nmap queries (it says the port is open, but -sV isn't identifying service any more).

So it looks like I broke it - and I'm currently waiting for the call to come in. In the meantime, I don't think my debug file is much use if the device itself is broken - unless it might come in handy to handle the issue?
